ykrt/src/compile/jitc_yk/aot_ir.rs

```diff
@@ -302,9 +303,9 @@ impl Operand {
         }
     }
 
-    /// Return the `InstructionID` of a local variable operand. Panics if called on other kinds of
-    /// operands.
-    pub(crate) fn to_instr_id(&self) -> InstructionID {
+    /// Return a reference to the instruction iff this operand is a local variable operand. Panics
```

Comment:
Hrm, not sure about this. "Reference" is too overloaded.

"Return a reference" is easily misunderstood to mean return a Rust reference.

A bit of background on the existing name. Ideally it'd be `InstrIdx`, but:
 - a) we already have a type of this name, which is an index local to a basic block.
 - b) it's not really just an index, is it? it's a triple of indices.

Question: What's up with `InstructionID`?

Reply:
I don't mind a renaming, but `Ref` and "reference" do more harm than good I feel. Sorry.